examples/go/selfservice/login/main.go

```diff
@@ -33,7 +35,7 @@ func performLogin() *ory.SuccessfulSelfServiceLoginWithoutBrowser {
 		ory.SubmitSelfServiceLoginFlowWithPasswordMethodBodyAsSubmitSelfServiceLoginFlowBody(&ory.SubmitSelfServiceLoginFlowWithPasswordMethodBody{
 			Method:             "password",
 			Password:           password,
-			PasswordIdentifier: &email,
+			PasswordIdentifier: pointerx.String(email),
```

Comment:
Looks like the merge history got a bit messed up? This commit seems unrelated to this PR

Reply:
hi yes I'll remove that

Reply:
revert